docs/guides/themeable-to-emotion-migration-guide.md

```diff
@@ -0,0 +1,459 @@
+- [Component Migration steps](#themeable-to-emotion-migration-guide/#component-migration-steps)
+  - [Refactor theme.js](#themeable-to-emotion-migration-guide/#component-migration-steps-refactor-theme.js)
+  - [Create the styles.js file](#themeable-to-emotion-migration-guide/#component-migration-steps-create-the-styles.js-file)
+  - [Make changes in the component](#themeable-to-emotion-migration-guide/#component-migration-steps-make-changes-in-the-component)
+  - [Example Component](#themeable-to-emotion-migration-guide/#component-migration-steps-example-component)
+  - [Theme tests](#themeable-to-emotion-migration-guide/#component-migration-steps-theme-tests)
+
+### The emotion package
+
+The `@instructure/emotion` package contains our implementation of the emotion theming library. You can find the full documentation [here](#emotion).
+
+Just like [**ui-themeable**](#https://instructure.github.io/instructure-ui-legacy-docs/#ui-themeable), **emotion** is used by:
+
+- adding a decorator to our themeable components,
+- mapping the global theme variables to component variables (in the `theme.js` file),
+- applying styles using the component variables (instead of `styles.css` file, now we use a `styles.js` file),
+- and applying themes by wrapping the component in a theme provider component.
+
+```javascript
+---
+embed: true
+---
+<ToggleBlockquote
+  summary="Migrating from ui-themeable"
+>
+  <ToggleBlockquote.Paragraph>
+    The usage is very similar, but migrating will require some refactoring, converting <code>css</code> to <code>js</code> and moving styling logic from the components to the <code>styles.js</code> file.
+  </ToggleBlockquote.Paragraph>
+  <ToggleBlockquote.Paragraph>
+    This guide will explain every step in detail, showing many examples, but <strong>feel free to contact the InstUI team</strong> if you need further help.
+  </ToggleBlockquote.Paragraph>
+</ToggleBlockquote>
+```
+
+## ApplyTheme to EmotionThemeProvider
+
+We made changes in how we apply themes too. You no longer need to register themes and use `ApplyTheme.generateTheme`. Instead, you can directly import them from `@instructure/ui-themes` (or your own themes) and pass them to the theme provider.
+
+The new theme provider is called `EmotionThemeProvider`, which is our own wrapper component for Emotion's [ThemeProvider](https://emotion.sh/docs/theming#themeprovider-reactcomponenttype).
+
+You can find detailed information about how it works and how to add theme overrides on the [EmotionThemeProvider](#EmotionThemeProvider) page.
+
+```js
+// before
+ReactDOM.render(
+  <ApplyTheme theme={ApplyTheme.generateTheme('instructure')}>
+    <App />
+  </ApplyTheme>,
+  element
+)
+
+// after
+import { instructure } from '@instructure/ui-themes'
+
+ReactDOM.render(
+  <EmotionThemeProvider theme={instructure}>
+    <App />
+  </EmotionThemeProvider>,
+  element
+)
+```
+
+## Component Migration steps
```

Comment:
Maybe write here Custom Component migration steps?

Reply:
Added a few lines of text to explain, also changed the title to "Migrating your @themeable components"